static/src/stylesheets/module/experiments/_embed.scss

```diff
@@ -0,0 +1,171 @@
+    height: 100%;
+    width: 100%;
+    border-top: 1px solid #4bc6df;
+    background-color: #efefec;
+
+    p {
+        line-height: 24px;
+        font-family: 'Guardian Egyptian Web';
+        padding-left: 5px;
+        padding-right: 15px;
+        margin-top: 8px;
+        margin-bottom: 8px;
+
+
+    }
+
+    .form-section {
+        font-family: 'Guardian Egyptian Web';
+        font-size: 1em;
+        line-height: 20px;
+        margin-top: 8px;
+        margin-bottom: 16px;
+    }
+
+
+
+    .title {
+        margin-top: 0;
+        margin-bottom: 2px;
+        padding-left: 5px;
+        padding-top: 7px;
+        font-family: 'Guardian Egyptian Web';
+        font-weight: bold;
+        font-size: 1.17em;
+    }
+
+
+    @mixin circular {
+        border-radius: 1000px; // Android 2.3 cannot deal with '50%'
+    }
+
+    @mixin button {
+        @include fs-textSans(4);
+        @include circular;
+
+        font-weight: bold;
+        text-decoration: none;
+        cursor: pointer;
+        text-align: left;
+        display: inline-block;
+        vertical-align: bottom;
+        outline: none;
+
+        padding: 6px $gs-gutter/2;
+        border: 0;
+        background-color: #ffffff;
+
+        &:hover {
+            background-color: $comment-support-1;
+            color: $white;
+        }
+
+        &.active {
+            background-color: $contribute-yellow;
+            color: $black;
+        }
+    }
+
+    .option-button {
+        @include button();
+
+        width: calc((100% - 26px) /4);
+        text-align: center;
+        margin-left: 3px;
+        margin-right: 0;
+        padding-top: 7px;
+        padding-bottom: 7px;
+    }
+
+    .button-group {
+        margin-right: 5px;
+        margin-left: 2px;
+    }
+
+    .flex-container {
+        display: flex;
+        flex-direction: column;
+        padding: 0 0 16px;
+        margin: 0;
+        list-style: none;
+
+
+    }
+
+    .amount-field {
+        display: flex;
+        flex-direction: column;
+
+    }
+
+
+    .contribute {
+        width: 130px;
+        background-color: $gu-blue;
+        color: #ffffff;
+        font-weight: bold;
+        margin-top: 12px;
+        margin-left: 5px;
+        margin-right: 7px;
+        text-align: left;
+        padding-left: 15px;
+        padding-right: 0;
+        &:hover {
+            background-color: $gu-blue-dark;
+        }
+
+        &:after {
+            content: '';
+
+            display: inline-block;
+            background-repeat: no-repeat;
+            width: 30px;
+            height: 20px;
+            background-image: svg-url('<svg xmlns="http://www.w3.org/2000/svg" width="40" height="40" viewBox="0 0 40 40"><path fill="white" stroke="white" d="M22.8 14.6L15.2 7l-.7.7 5.5 6.6H6v1.5h14l-5.5 6.6.7.7 7.6-7.6v-.9"/></svg>');
+            background-size: 100%;
+            float: right;
+        }
+    }
+
+
+    @include mq($from: phablet) {
+        .bottomClass.option-button {
```

Comment:
I think what you're doing is replacing the BEM way of doing modifiers with a changing classes way - you shouldn't need to pass loads more parameters into the template, you can do instead of:

```
<div class="amount-field <%=bottomClass%>">
                 <div>
                     <div class="js-button-group button-group u-cf">
                         <button type="button" tabindex="8" class="js-button option-button option-button--bold <%=bottomClass%>" 
```

this: 

```
<div class="amount-field amount-field--<%=bottomClass%>">
                 <div>
                     <div class="js-button-group button-group u-cf">
                         <button type="button" tabindex="8" class="js-button option-button option-button--bold option-button--<%=bottomClass%>" 
```

And style on `.option-button--bottom` (probably don't need the word class and classes should be hyphenated btw) and `.amount-field--bottom`

Does that make sense?